lib/utils.ts

```diff
@@ -0,0 +1,28 @@
+type Port = string | number;
+
+export const requestFormatter = req => res => {
+  if (req.query.q) {
+    res.query = req.query.q;
+    if (res.query.slice(-1) != '?') {
+      res.query = res.query + '?';
+    }
+  }
+  return { ...res };
+};
+
+export const dilate = req => s => res => {
+  const { response } = res;
+
+  return req.query.big == 1
+    ? {
+        ...res,
+        response: `${response.substr(0, 3)}${s}${response.substr(
+          3,
+          response.length - 1
+        )}`,
+      }
+    : { ...res };
+};
+
+export const getPort = (envPort: Port = 3000) =>
+  typeof envPort === 'string' ? Number(envPort) : envPort;
```

Comment:
What if `Number(envPort)` fails? Are you fine with this to throw?

Reply:
@balanza thanks for reviewing! Is there a way to avoid that kind of stuff using TS feats?